test/unit/test_tpm2_session.c

```diff
@@ -211,11 +212,15 @@ static void test_tpm2_session_defaults_good(void **state) {
     assert_int_equal(auth_hash, TPM2_ALG_SHA256);
 
     tpm2_session_free(&s);
+    free (e);
```

Comment:
I think we would need to use the test functions that have a setup and teardown, as any assert() will not run the cleanup code. just put this pointer in *state and pass it through.

Something changed is causing a seg fault in the travis logs.